.pylintrc

```diff
@@ -422,7 +423,10 @@ disable=raw-checker-failed,
         useless-suppression,
         deprecated-pragma,
         use-symbolic-message-instead,
-        logging-fstring-interpolation
+        logging-fstring-interpolation,
```

Comment:
Thanks for the additional details. I would recommend we get rid of the `consider-using-f-string` ignore, since this is a good check to ensure we do not backslide into old python-formatted strings. As you point out,  `logging-fstring-interpolation` is already ignored, so no one should be confused by conflicting messages.

Reply:
Okay. Sounds good. I'll make the mod to this list, and the code so it passes the linter.